docs/vm-images.md

```diff
@@ -1,10 +1,31 @@
 # Supported and Tested VM Images
 
-* [HPC CentOS 7 VM Image](#hpc-centos-7-vm-image)
-* [Ubuntu](#ubuntu)
+* [HPC Rocky Linux 8](#hpc-rocky-linux-8)
+* [HPC CentOS 7](#hpc-centos-7)
+* [Debian 11](#debian-11)
+* [Ubuntu 20.04 LTS](#ubuntu-2004-lts)
 * [Other Images](#other-images)
 
-## HPC CentOS 7 VM Image
+## HPC Rocky Linux 8
+
+HPC Rocky Linux 8 is the primary supported VM image for HPC workloads on Google Cloud.
```

Comment:
It is planned to be the primary image in 2024, I would not say it is currently the primary image since we don't do most of our integration tests with it.

Reply:
Fixed.